src/main/java/se/sundsvall/emailsender/api/EmailResource.java

```diff
@@ -2,22 +2,29 @@
 
 import static org.springframework.http.ResponseEntity.ok;
 
+import jakarta.mail.MessagingException;
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.Valid;
+
 import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 import org.zalando.problem.Problem;
 
+import se.sundsvall.emailsender.api.model.SendEmailRequest;
+import se.sundsvall.emailsender.service.EmailService;
+
 import io.swagger.v3.oas.annotations.Operation;
 import io.swagger.v3.oas.annotations.media.Content;
 import io.swagger.v3.oas.annotations.media.Schema;
 import io.swagger.v3.oas.annotations.responses.ApiResponse;
-import jakarta.mail.MessagingException;
-import jakarta.validation.Valid;
-import se.sundsvall.emailsender.api.model.SendEmailRequest;
-import se.sundsvall.emailsender.service.EmailService;
+import io.swagger.v3.oas.annotations.tags.Tag;
 
 @RestController
+@Tag(name = "Email resources", description = "Resources for sending emails")
+@Validated
```

Comment:
This isn't required, since you have the `@Valid` annotation on the request parameter and also since it's main usage is for validation groups, which isn't used/required here